packages/client/wallets/smart-wallet/src/utils/log.ts

```diff
@@ -1,47 +1,10 @@
-import { logError, logInfo } from "../services/logging";
-import { SCW_SERVICE } from "./constants";
+import { SDKLogger } from "@crossmint/client-sdk-base";
 
-export async function logPerformance<T>(name: string, cb: () => Promise<T>, extraInfo?: object) {
-    const start = new Date().getTime();
-    const result = await cb();
-    const durationInMs = new Date().getTime() - start;
-    const args = { durationInMs, ...extraInfo };
-    logInfo(`[${SCW_SERVICE} - ${name} - TIME] - ${beautify(args)}`, { args });
-    return result;
-}
-
-function beautify(json: any) {
-    try {
-        return json != null ? JSON.stringify(json, null, 2) : json;
-    } catch (error) {
-        return stringifyAvoidingCircular(json);
-    }
-}
-
-function stringifyAvoidingCircular(json: any) {
-    // stringify an object, avoiding circular structures
-    // https://stackoverflow.com/a/31557814
-    const simpleObject: { [key: string]: any } = {};
-    for (const prop in json) {
-        if (!Object.prototype.hasOwnProperty.call(json, prop)) {
-            continue;
-        }
-        if (typeof json[prop] == "object") {
-            continue;
-        }
-        if (typeof json[prop] == "function") {
-            continue;
-        }
-        simpleObject[prop] = json[prop];
-    }
-    return JSON.stringify(simpleObject, null, 2); // returns cleaned up JSON
-}
-
-export function errorToJSON(error: Error | unknown) {
```

Comment:
This function is redundant, feel free to delete it too if you'd like.

Only used in one spot, and the spot it's used would be better served by `error.message`